libcxx/test/libcxx/localization/locales/locale.convenience/conversions/conversions.string/ctor_move.pass.cpp

```diff
@@ -10,7 +10,7 @@
 
 // UNSUPPORTED: c++03
 
-// ADDITIONAL_COMPILE_FLAGS: -D_LIBCPP_DISABLE_DEPRECATION_WARNINGS
+// ADDITIONAL_COMPILE_FLAGS: -D_LIBCPP_DISABLE_DEPRECATION_WARNINGS -D_LIBCPP_ENABLE_CXX26_REMOVED_CODECVT
```

Comment:
Not attached to this line: perhaps we could add a test that does

```
#include <codecvt>

void f() {
  std:: codecvt_utf8<whatever> x; // expected-error {{no member named codecvt_utf8 in namespace std}}
}
```

or something like that?